src/transformers/eagerSourceMapTransformer.ts

```diff
@@ -44,10 +48,10 @@ export class EagerSourceMapTransformer extends BaseSourceMapTransformer {
 
     private discoverSourceMapForGeneratedScript(generatedScriptPath: string): Promise<void | SourceMap> {
         return this.findSourceMapUrlInFile(generatedScriptPath)
-            .then(uri => {
+            .then(async uri => {
                 if (uri) {
                     logger.log(`SourceMaps: sourcemap url parsed from end of generated content: ${uri}`);
-                    return this._sourceMaps.processNewSourceMap(generatedScriptPath, uri, this._isVSClient);
+                    await this._sourceMaps.processNewSourceMap(generatedScriptPath, uri, this._isVSClient);
```

Comment:
It looks like this wants to return the SourceMap?

Reply:
We are currently not using this result value. I changed this method to return Promise<void> so this won't be confusing